src/main/scala/it/pagopa/interop/authorizationprocess/api/impl/ClientApiServiceImpl.scala

```diff
@@ -940,11 +937,16 @@ final case class ClientApiServiceImpl(
     )
   }
 
-  /** Code: 200, Message: returns the corresponding base 64 encoded key, DataType: EncodedClientKey
-    * Code: 401, Message: Unauthorized, DataType: Problem
-    * Code: 404, Message: Key not found, DataType: Problem
-    * Code: 500, Message: Internal Server Error, DataType: Problem
-    */
+  private[this] def checkAuthorization(clientId: UUID)(implicit contexts: Seq[(String, String)]): Future[Unit] =
```

Comment:
what about `isConsumer`/`assertConsumer` or similar?
so it's clear what the function is checking

Reply:
makes sense